src/main/java/com/zenjava/javafx/maven/plugin/JarMojo.java

```diff
@@ -249,22 +277,54 @@ public void execute() throws MojoExecutionException, MojoFailureException {
                 File dest = new File(libDir, artifactFile.getName());
                 if( !dest.exists() ){
                     try{
-                        Files.copy(artifactFile.toPath(), dest.toPath());
+                        if( !skipCopyingDependencies ){
+                            Files.copy(artifactFile.toPath(), dest.toPath());
+                        } else {
+                            getLog().info(String.format("Skipped copying classpath element: %s", artifactFile.getAbsolutePath()));
+                        }
                     } catch(IOException ex){
                         getLog().warn(String.format("Couldn't read from file %s", artifactFile.getAbsolutePath()));
                         getLog().debug(ex);
                         brokenArtifacts.add(artifactFile.getAbsolutePath());
                     }
                 }
-                classpath.append("lib/").append(artifactFile.getName()).append(" ");
+                classpath.append(libFolderName).append("/").append(artifactFile.getName()).append(" ");
             });
             if( !brokenArtifacts.isEmpty() ){
                 throw new MojoExecutionException("Error copying dependencies for application");
             }
         } catch(IOException e){
             throw new MojoExecutionException("Error copying dependency for application", e);
         }
-        createJarParams.setClasspath(classpath.toString());
+
+        if( useLibFolderContentForManifestClasspath ){
+            StringBuilder scannedClasspath = new StringBuilder();
+            try{
+                Files.walkFileTree(libDir.toPath(), new SimpleFileVisitor<Path>() {
+                    @Override
+                    public FileVisitResult visitFile(Path file, BasicFileAttributes attrs) throws IOException {
+                        scannedClasspath.append(file.toString().replace("\\", "/")).append(" ");
```

Comment:
Hm ... it should be relative, but it is missing the libDirName ;) too tired yesterday, will fix that with 8.8.1